Makefile

```diff
@@ -229,7 +227,7 @@ protobuf:
 .PHONY: .go-version
 .go-version:
 	@actual=$$($(GO) version); \
-	echo "$${actual}" | grep -q $(GOVERS) || \
+	echo "$${actual}" | grep -q '\b$(GOVERS)\b' || \
```

Comment:
I don't think we want to be this strict in most cases. An exact match is good for release builds but there's no reason to require that all developers upgrade to go 1.6.1 in lockstep when it is released.